Storage/SQL/SQLiteHistory.swift

```diff
@@ -82,18 +82,30 @@ public class SQLiteHistory {
 
     private var ignoredSchemes = ["about"]
 
-    required public init(db: BrowserDB) {
+    required public init?(db: BrowserDB, version: Int? = nil) {
```

Comment:
If you're just going to `!` this at the call sites, is there a point in making this optional?